javascript-bank/bank.js

```diff
@@ -0,0 +1,37 @@
+/* exported Bank */
+function Bank() {
+  this.accounts = [];
+  this.nextAccountNumber = 1;
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if ((balance > 0) && (balance % 1 === 0)) {
+    let acc = new Account(balance, holder);
+    acc.nextAccountNumber = this.nextAccountNumber;
+    acc.balance = balance;
+    this.accounts.push(acc);
+    this.nextAccountNumber +=1;
+    return (this.nextAccountNumber - 1)
+  } else {
+    return null
+  }
+};
+
+Bank.prototype.getAccount = function (number) {
+  let account = null;
+  for (let acc of this.accounts) {
```

Comment:
Can you explain how `for ... of` loops work? If so, great! If not, use something more basic.